.github/workflows/main.yml

```diff
@@ -0,0 +1,32 @@
+name: LSDB
+
+on:   
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ["3.8", "3.9", "3.10"]
+
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v4
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install numpy pytest pytest-cov
+          pip install .[all]
+          if [ -f requirements.txt ]; then pip install -r requirements.txt; fi          
+          source setup.bash          
```

Comment:
Just curious - is `source setup.bash` needed here? I looked at the setup.bash file, and it's modifies PYTHONPATH. But was that modification originally so that you could run tests locally in your src directory while developing? And if so, is it needed here once the code is pip installed?

Reply:
I have the same question as Drew about setup.bash. Other than that it looks good to me.

Reply:
Oh hey! It worked that time!

I've tried a handful of different things, but I guess not this particular incantation. I'm happy to remove that line!